package.json

```diff
@@ -73,13 +85,16 @@
     "pug": "2.0.4"
   },
   "devDependencies": {
+    "@types/node": "*",
     "@types/recursive-readdir": "2.2.0",
     "@types/semver": "6.0.1",
     "@types/sinon": "7.0.13",
     "@types/slug": "0.9.1",
     "@typescript-eslint/eslint-plugin": "1.11.0",
     "@typescript-eslint/parser": "1.11.0",
     "ava": "2.1.0",
+    "can-npm-publish": "^1.3.1",
+    "del-cli": "^2.0.0",
```

Comment:
Please pin the dependencies version.
If you do not pin it, [it will be reported from the Renovate bot](https://github.com/sounisi5011/metalsmith-pug-extra/pull/35).